code/datums/components/transforming.dm

```diff
@@ -51,6 +55,8 @@
 	hitsound_on = 'sound/weapons/blade1.ogg',
 	w_class_on = WEIGHT_CLASS_BULKY,
 	clumsy_check = TRUE,
+	clumsy_damage_type = BRUTE,
+	clumsy_damage = 10,
```

Comment:
I don't think we need two new args for this already lengthy arg list, why not just inherit the force (maybe with a modifier, like 0.5) and damage type of the weapon itself?

Reply:
Damage type sure, but for the actual amount of damage, I still think it should be adjustable.